app/helpers/application_helper.rb

```diff
@@ -24,6 +24,14 @@ def wordpress_path(path = '')
     Settings.wordpress.url.chomp('/') + '/' + path.sub(/^\/+/) { $1 }
   end
 
+  ##
+  # Returns frontend path with correctly joining '/'
+  # @param path String (with or without leading slash)
+  # @return String
+  def api_path(path = '')
+    Settings.api.url.chomp('/') + '/' + path.sub(/^\/+/) { $1 }
```

Comment:
I think for now it's fine if it's `http`. We could also add another `sameAs` with the `https` URI if we were really concerned about it.